permissions.json

```diff
@@ -0,0 +1,13 @@
+{
+	"Version": "2012-10-17",
+	"Statement": [{
+		"Effect": "Allow",
+		"Action": [
+			"logs:CreateLogStream",
```

Comment:
we need permission to allow the logs to set retention time. We also need to complete this permissions json to create a new permission. Right now, I created a new permission from the AWS console, called "[eks-fargate-logging-policy](https://us-east-1.console.aws.amazon.com/iam/home#/policies/arn:aws:iam::242905224710:policy/eks-fargate-logging-policy)" and manually attached it to the Fargate pod execution roles.

Reply:
After the role is created properly, this role created by the console needs to be deleted